package.json

```diff
@@ -75,7 +75,7 @@
     "@vitest/coverage-istanbul": "2.0.5",
     "@vitest/ui": "2.0.5",
     "cross-env": "^7.0.3",
-    "electron": "^32.0.0",
+    "electron": "^31.0.0",
```

Comment:
~31.0.0

Reply:
it should be fine to be any version 31+ but less than 32. so ^ should be good